test/tools/unified-spec-runner/operations.ts

```diff
@@ -510,7 +510,7 @@ operations.set('getKey', async ({ entities, operation }) => {
 operations.set('getKeys', async ({ entities, operation }) => {
   const clientEncryption = entities.getEntity('clientEncryption', operation.object);
 
-  return clientEncryption.getKeys();
+  return clientEncryption.getKeys().toArray();
```

Comment:
`getKeys` returns a cursor, missed in the initial implementation of these operations in the unified runner